cli/info/info.go

```diff
@@ -44,9 +47,35 @@ func infoMain(args []string, c cli.Config) (err error) {
 	if err != nil {
 		return err
 	}
+
+	_, err = helpers.ParseCertificatePEM(certPem)
+	if err != nil {
+		return errors.Wrap(errors.CertificateError, errors.DecodeFailed, err)
+	}
+
+	s, err := sign.SignerFromConfig(c)
```

Comment:
This is questionable for remote signer. I understand this is confusing because it is the frontier of development.
A remote signer's profile has no actual signing policy, it only has auth key config and a remote address. So getting usage and expiry string from that would just be zero values.
Although the cli command parameter '-config' is included, the purpose is to provide the auth key to generate auth token. But, right now the authentication is not yet turned on. Since the info endpoint is currently just returning a certificate, it is considered safe to reveal anyway. So for now, let's ignore '-config'

The real problem is, if api/info starts to return more information, the api/client need to adopt to the changes, at line 46, the certificate is returned by api/client, and take a look at api/client, we realize the result is actually a map, but api/client.Server captures the value under 'certificate' and return it. So change api/client to actually return the true response of api info, so the usage and expiry string should be just two lookups.